ios/PromotedMetricsModule.m

```diff
@@ -82,16 +82,18 @@ @interface RCT_EXTERN_REMAP_MODULE(PromotedMetrics, PromotedMetricsModule, NSObj
                   useCase:(NSInteger)useCase)
 
 #pragma mark - ImpressionLogger
-RCT_EXTERN_METHOD(collectionViewDidLoad:(NSString *)collectionViewName)
+RCT_EXTERN_METHOD(collectionViewDidMount:(NSString *)collectionViewName)
```

Comment:
have you looked into using some lexer or something to determine method name consistency at the code locations that need it?

Reply:
The tests should cover this. I can think of a solution that improves the macro, maybe I'll suggest it to the RN community.